subcommittees/mentoring/procedure-debriefing-session.md

```diff
@@ -0,0 +1,109 @@
+
+## Terminology
+
+-   **SCF**: Software Carpentry Foundation.
+
+-   **DC**: Data Carpentry.
+
+-   **Debriefing session**: is an online meeting
+    where instructors could share experience about the workshops
+    they recently taught.
+
+-   **Facilitator**: is one member of the Mentoring Subcommittee
+    that attend the debriefing session to coordinate it
+    and write one blog post about it.
+
+## We Need a Debriefing Session
+
+1.  SCF and DC are community based project
+    and the community need opportunity to interact.
+
+2.  SCF and DC believe in jugyokenkyu, or "lesson study",
+    and evidence-based education
+    so instructors need opportunity to exchange experience.
+
+## Let's do a Debriefing Session
+
+The Mentoring Subcommittee is responsible for organizing
+the debriefing sessions so that
+
+-   each month has at least one debriefing session,
+
+-   instructors are notify with at least one week in advance and
+
+-   the session has a facilitator to conduct the discussion.
+
+## Inviting Instructors
+
+The dates of the debriefing sessions are listed
+on SCF community calendar,
+https://calendar.google.com/calendar/embed?src=oseuuoht0tvjbokgg3noh8c47g%40group.calendar.google.com.
+
+Invitation should be send directly to instructors
+that taught between the last and the next debriefing session by email.
+The contact address of the instructors are available at
+https://amy.software-carpentry.org/workshops/debrief.
+
+## Show time
+
+Each debriefing session **must** have at least one **facilitator**.
+
+The **facilitator** is responsible to provide one place
+where instructors attending the debriefing could share notes
+of the debriefing.
+
+The **facilitator** is also responsible
+to help instructors answer the questions during the session.
+
+## Questions
+
+1.  Public
+
+    1.  How many learners attend the workshop?
+
+    2.  Big changes in the number of learners during the workshop?
+
+    3.  Learners had any installation problem with the tools
+        requested that they installed?
+
+2.  Content
+
+    1.  What was actually taught?
+
+        If a SCF workshop:
+        Shell, Git, Python, R, MATLAB or SQL?
+
+        If a DC workshop:
+        Biology or Genomics?
+        R, Python, Spreadsheet caveats, Open Refine, SQL, Shell,
+        Cloud computing intro?
+
+    2.  How much time was spent on each topic?
+
+    3.  Did you adapt the lesson material or use it as written?
+
+3.  Feedback
+
+    1.  What worked well that we should do again?
+
+    2.  What went wrong / fell flat / shouldn't we ever, ever do again?
+
+4.  Extra
+
+    1.  Anything else that might be interesting?
+
+    2.  Who were helpers, and their contact info?
+        (especially those not listed on course website)
+
+## Report to the Community
+
+After each debriefing session,
+the **facilitator** of that debriefing session **must**
+write a blog post summarizing the topics discussed.
+In some cases, the facilitator **should**
```

Comment:
I think it should be the responsibility of the facilitator to encourage the attendees to file the issues. The person raising an issue or problem is in the best position to raise an issue and engage in discussions about the specifics.